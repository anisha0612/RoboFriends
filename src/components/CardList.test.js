import React from "react";
import { create } from "react-test-renderer";
import CardList from "./CardList";

it("expect to render Card component using snapshot testing", () => {
  const mockRobots = [
    {
      id: 1,
      name: "John Doe",
      email: "john.doe@gmail.com",
    },
    {
      id: 2,
      name: "Jane Doe",
      email: "jane.doe@gmail.com",
    },
  ];
  const cardList = create(<CardList robots={mockRobots} />);
  expect(cardList.toJSON()).toMatchSnapshot();
});
