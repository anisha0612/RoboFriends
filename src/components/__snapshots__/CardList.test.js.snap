// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card component using snapshot testing 1`] = `
<div>
  <div
    className="tc br4 dib bg-light-yellow grow ma3 pa3 shadow-4"
  >
    <img
      alt="roboPhoto"
      src="https://robohash.org/1?size=200x200"
    />
    <div>
      <h2>
        John Doe
      </h2>
      <p>
        john.doe@gmail.com
      </p>
    </div>
  </div>
  <div
    className="tc br4 dib bg-light-yellow grow ma3 pa3 shadow-4"
  >
    <img
      alt="roboPhoto"
      src="https://robohash.org/2?size=200x200"
    />
    <div>
      <h2>
        Jane Doe
      </h2>
      <p>
        jane.doe@gmail.com
      </p>
    </div>
  </div>
</div>
`;
